### PR TITLE
Local dai migration for testnets

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: migrations/3_configure.js [hunk 1]
```diff
@@ -34,7 +49,7 @@ async function deploy(options, accounts) {
   );
 
   // GSN TESTING!
-  await patronageERC20.addMinter(accounts[0]);
+  // await patronageERC20.addMinter(accounts[0]);
```
Comment (JasoonS) [inline migrations/3_configure.js:52]: Only needed for testing

File: migrations/3_configure.js [hunk 2]
```diff
@@ -67,7 +82,7 @@ async function deploy(options, accounts) {
     // address _steward,
     steward.address,
     // address _token
-    patronageERC721.address,
+    patronageERC20.address,
```
Comment (JasoonS) [inline migrations/3_configure.js:85]: This bug took me an hour and a half to find :see_no_evil: Sorry if you ran into this too @DenhamPreen

File: migrations/4_launch_tokens.js [hunk 3]
```diff
@@ -1,11 +1,60 @@
...
+const Dai = artifacts.require("./Dai.sol");
+
+const { ether } = require("@openzeppelin/test-helpers");
+const { daiPermitGeneration } = require("../test/helpers");
 
 const testAccountAddress = "0x8c7A88756EbbF46Ede65E4D678359cAC5f08f7b2";
 
 const starHarbergerTaxRateAnimal = "240" + "0000000000"; // Harberger tax rate of 240% per year
 
+const buyAuctionPermit = async (
```
Comment (JasoonS) [inline migrations/4_launch_tokens.js:12]: Couldn't get this function to work, very frustraiting... Will let Marvin loose on this :)

File: migrations/4_launch_tokens.js [hunk 4]
```diff
@@ -36,9 +85,109 @@ module.exports = function(deployer, networkName, accounts) {
       ],
       [],
       [],
-      [1598186000, 1598186000, 1598186000, 1598186000],
+      [0, 0, 0, 0],
```
Comment (JasoonS) [inline migrations/4_launch_tokens.js:88]: This change wasn't necessary, but whatever...

File: migrations/3_configure.js (hunk 2 above)
Comment (DenhamPreen) [reply migrations/3_configure.js:85]: Haha I actually got stuck on 2_deploy.js

**Pull request closed without merging**